runtime/opt/taupage/init.d/02-register-td-agent.py

```diff
@@ -152,6 +152,21 @@ def update_configuration_from_template():
     if not logging_config:
         logger.info('Found no logging section in senza.yaml')
         raise SystemExit()
+    else:
+        try:
```

Comment:
using a file context manager is a better pattern

Reply:
good point, changed